File: app/src/main/java/com/dantsu/thermalprinter/data/Comment.java
package com.dantsu.thermalprinter.data;

import com.fasterxml.jackson.annotation.JsonProperty;

import java.util.List;

public class Comment {
    @JsonProperty("comment")
    public String comment;
    @JsonProperty("uniqueId")
    public String uniqueId;
    @JsonProperty("userId")
    public String userId;
    @JsonProperty("nickname")
    public String nickname;
    @JsonProperty("secUid")
    public String secUid;
    @JsonProperty("followRole")
    public int followRole;
    @JsonProperty("userBadges")
    public List<Object> userBadges;
    @JsonProperty("profilePictureUrl")
    public String profilePictureUrl;
    @JsonProperty("userDetail")
    public UserDetail userDetail ;
}

File: app/src/main/java/com/dantsu/thermalprinter/data/Comment.kt
package com.dantsu.thermalprinter.data

import com.fasterxml.jackson.annotation.JsonProperty

class Comment {
    @JvmField
    @JsonProperty("comment")
    var comment: String? = null

    @JvmField
    @JsonProperty("uniqueId")
    var uniqueId: String? = null

    @JsonProperty("userId")
    var userId: String? = null

    @JvmField
    @JsonProperty("nickname")
    var nickname: String? = null

    @JsonProperty("secUid")
    var secUid: String? = null

    @JsonProperty("followRole")
    var followRole = 0

    @JsonProperty("userBadges")
    var userBadges: List<Any>? = null

    @JvmField
    @JsonProperty("profilePictureUrl")
    var profilePictureUrl: String? = null

    @JsonProperty("userDetail")
    var userDetail: UserDetail? = null
}